ccm_web/client/Dockerfile.dev

```diff
@@ -0,0 +1,12 @@
+FROM node:14-slim AS node
+WORKDIR /app/
```

Comment:
is this file being used?

Reply:
Yes, by `docker-compose` for the development set up.